ansible/roles/nginx/templates/node.conf.j2

```diff
@@ -40,4 +40,12 @@ server {
 
         proxy_pass   http://localhost:{{ django_port }};
     }
+
+    location /couchdb {
+        rewrite /couchdb/(.*) /$1 break;
+        proxy_pass http://localhost:5984;
```

Comment:
yes and no... couch_listen_address doesn't specify the port, it specifies the ip address. Since we proxy couch behind nginx, it never needs to be able to listen on an external port, it always listens on localhost, and nginx proxies to that. before it was sitting behind nginx, we allowed access to it in vagrant but not production, which was why we needed that flexibility